docs/en/UFSExtensions.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: global
+group: Under Store
+title: Under Storage Extensions
+---
+
+* Table of Contents
+{:toc}
+
+This page is intended for users of under storage extensions. Please look at [developing
+extensions](DevelopingUFSExtensions.html) for an extension development guide.
+
+Alluxio can be extended with the addition of under storage modules at runtime. Under storage
+extensions (built as JARs) can be included at a specific location to be picked up by core Alluxio
+without the need to restart any running processes. Adding new under storage connectors to Alluxio
+can be used to enable Alluxio to work with your choice of storage system.
```

Comment:
to point out that in this way, users can connect Alluxio to a UFS even without the official support from Alluxio